roles/openshift_ovirt/tasks/create_inventory_from_vms.yaml

```diff
@@ -20,23 +23,23 @@
 - name: Add infra Vms to the nodes inventory group
   add_host:
     groups: nodes
-    hostname: '{{ [item.name, item.count - 1, ".", openshift_ovirt_dns_zone] | join }}'
+    hostname: '{{ (item.name, (item.count > 1) | ternary((("[0:",item.count - 1 , "]") | join), "0"), ".", openshift_ovirt_dns_zone) | join }}'
     openshift_node_group_name: node-config-infra
     openshift_schedulable: true
   with_items: "{{ openshift_ovirt_vm_manifest }}"
   when: item.count > 0 and item.name == 'infra'
 
 - name: Add etcd Vms to the etcd inventory group
   add_host:
-    hostname: '{{ [item.name, item.count - 1, ".", openshift_ovirt_dns_zone] | join }}'
-    groups: 'etcd'
+    hostname: '{{ (item.name, (item.count > 1) | ternary((("[0:",item.count - 1 , "]") | join), "0"), ".", openshift_ovirt_dns_zone) | join }}'
+    groups: [etcd]
```

Comment:
Hmm, per the earlier comment I think these should be uniformly just the string.

Reply:
done